dms/migrations/13.0.1/post-migration.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
```

Comment:
Disable this linter that is not suitable for migrations:
```suggestion
from openupgradelib import openupgrade  # pylint: disable=W7936
```